src/server.ts

```diff
@@ -12,6 +12,6 @@ app.listen(PORT, () => {
 
   // Seed the database with some data
   if (environment === "dev") {
-    populateDummyData();
+    // populateDummyData();
```

Comment:
Now that my db pull request has been merged, we have a populateDummyData function, so this doess't need to be commented out.

Reply:
please can we all ensure whatever code that is pushed works and code lines should not cause issue once the next developer pulls